modules/shortcodes/flickr.php

```diff
@@ -123,12 +92,12 @@ function flickr_embed_to_shortcode( $content ) {
 function flickr_shortcode_handler( $atts ) {
 	$atts = shortcode_atts(
 		array(
-			'video'     => 0,
-			'photo'     => 0,
-			'show_info' => 0,
```

Comment:
Is there a specific reason why you removed this attribute?

Reply:
Yes, the `show_info` param was being used in the flash video output, but it's not needed in the `<video>` tag approach I took on this.